.devops/templates/deployE2E.yml

```diff
@@ -24,14 +24,23 @@ steps:
       yarn workspace @fluentui/pr-deploy-site generate:site
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31041&commitId=90f5f58fbe4f1a58a74407b54cb262a343ec12da&env=prod&clientType=FLUENTUI)


  <details><summary><b>RatingDisplay Converged</b> 18 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|RatingDisplay Converged.Neutral with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.custom circle icons.rest.chromium.png|31| Changed|
|RatingDisplay Converged.rounded down.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Marigold with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.Neutral with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Neutral with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.size small with value and count.rest.chromium.png|124| Changed|
|RatingDisplay Converged.size medium with value and count.rest.chromium.png|124| Changed|
|RatingDisplay Converged.Compact.rest.chromium.png|31| Changed|
|RatingDisplay Converged.rounded up.rest.chromium.png|70| Changed|
|RatingDisplay Converged.size extra-large with value and count.rest.chromium.png|261| Changed|
|RatingDisplay Converged.Brand with half value.rest.chromium.png|60| Changed|
|RatingDisplay Converged.Marigold with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.custom square icons.rest.chromium.png|31| Changed|
|RatingDisplay Converged.Brand with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Marigold with half value - Dark Mode.rest.chromium.png|59| Changed|
|RatingDisplay Converged.Brand with half value - High Contrast.rest.chromium.png|70| Changed|
|RatingDisplay Converged.size large with value and count.rest.chromium.png|166| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>